cmd/thanos/query-frontend.go

```diff
@@ -62,6 +66,9 @@ func (c *queryRangeConfig) registerFlag(cmd *kingpin.CmdClause) {
 	cmd.Flag("query-range.response-cache-max-freshness", "Most recent allowed cacheable result, to prevent caching very recent results that might still be in flux.").
 		Default("1m").DurationVar(&c.cacheMaxFreshness)
 
+	cmd.Flag("query-range.partial-response", "Enable partial response for queries if no partial_response param is specified. --no-query.partial-response for disabling.").
```

Comment:
I guest it should be `--no-query-range`, no?
```suggestion
	cmd.Flag("query-range.partial-response", "Enable partial response for queries if no partial_response param is specified. --no-query-range.partial-response for disabling.").
```

Reply:
Good catch! Let me update the pr.